app/helpers/blacklight/blacklight_helper_behavior.rb

```diff
@@ -79,7 +79,14 @@ def extra_body_classes
   # Render the search navbar
   # @return [String]
   def render_search_bar
-    search_bar_presenter.render
```

Comment:
I'm worried about the backwards-compatible implications of skipping the presenter here. I think downstream applications could have overridden that or the `catalog/search_form` partial.